### PR TITLE
US136321 - Switch to rems

### PR DESCRIPTION
**Pull request by svanherk**
Switch to use rems so when the tag size adjusts, the user-profile image adjusts as well.

### CODE REVIEW

File: test/d2l-profile-image-base.test.js [hunk 1]
```diff
@@ -1,40 +1,18 @@
 import '../components/d2l-profile-image-base.js';
 import { expect, fixture, html } from '@open-wc/testing';
 import { match, spy, stub } from 'sinon';
-
-const fixtures = {
```
Comment (svanherk) [inline test/d2l-profile-image-base.test.js:5]: This looks worst than it is - most of this is just cleanup, plus switching out the "instantiates with default properties" test for the constructor test, and then removing the sizes test.  That test is no longer needed now that we have the visual-diff tests.

File: test/d2l-profile-image.test.js [hunk 2]
```diff
@@ -1,67 +1,41 @@
 import '../components/d2l-profile-image.js';
 import { expect, fixture, html } from '@open-wc/testing';
+import { runConstructor } from '@brightspace-ui/core/tools/constructor-test-helper.js';
 import SirenParse from 'siren-parser';
 
-const fixtures = {
```
Comment (svanherk) [inline test/d2l-profile-image.test.js:5]: Same changes as the other test file (lots of repetition between the two)

File: components/d2l-profile-image-base.js [hunk 3]
```diff
@@ -77,40 +77,40 @@ class ProfileImageBase extends LitElement {
...
+					min-width: 1.5rem;
+					width: 1.5rem;
 				}
 				:host([size="small"]) .d2l-profile-image-container {
 					--d2l-profile-image-border-radius: 4px;
 				}
 
 				:host([size="medium"]) {
-					height: 42px;
+					height: 2.1rem;
```
Comment (dlockhart) [inline components/d2l-profile-image-base.js:90]: With our inputs and buttons, we arrive at `42px` from `2rem` for the text/padding + `2px` for the top/bottom border. So not sure if this should be `calc(2rem + 2px)`?

Comment (svanherk) [reply components/d2l-profile-image-base.js:90]: Yeah that could make sense, so they stay in line with button sizes.  Just for this size?  The slim buttons are just `1.5rem`, although they could be updated to follow a similar pattern.  And then not sure if the larger sizes are trying to match any other component heights.

Comment (dlockhart) [reply components/d2l-profile-image-base.js:90]: Yeah not totally sure... I don't think design has really been thinking of this stuff in terms of `rem`s with the base font size being dynamic -- so often they'll just put `42px` or `30px` or whatever but what they really mean is `2px` border and then `X rem`.

Comment (svanherk) [reply components/d2l-profile-image-base.js:90]: I'll do this one for now - sounded like Jeff wanted to rethink this when we move it to `core` to anyways, he was surprised it couldn't just be scaled to any size

**Review by dlockhart — APPROVED** [review]

**Pull request merged**